### DISCUSSION TITLE
How can I have file name first in buffers and old_files commands like files command?

### DISCUSSION THREAD

**Discussion by niamleeson** [ibhagwan/fzf-lua#1173] [Q&A] [answered]
In the options for files command, you can provide formatter to show filename first. How can you do this for buffers and old_files commands?

**Comment by ibhagwan** [comment]
Buffers isn’t supported yet (which I’m planing to add as well as the custom highlight for the folder part), for old files just add `formatter=path.filename_first`.

**Reply by niamleeson** [reply]
That'd be awesome! Thank you for being so responsive!

**Reply by ibhagwan** [reply, marked as answer]
@niamleeson, try the latest commit https://github.com/ibhagwan/fzf-lua/commit/2702816ef910ea956b3154ce1e3cbd02b2a6c127, added support for the formatter for buffer type entries (`buffers`, `tabs`):
```lua
:FzfLua buffers formatter=path.filename_first
```

**Reply by niamleeson** [reply]
Woooow so fast! Thank you so much! Really appreciate it!

**Reply by ibhagwan** [reply]
@niamleeson, see #1175 for further customization of the directory part highlight group.

**Reply by niamleeson** [reply]
Thank you for additional help! You are awesome!
